source/System.cpp

```diff
@@ -1068,7 +1096,7 @@ void System::UpdateNeighbors(const Set<System> &systems, double distance)
 	{
 		const System &other = it.second;
 		// Skip systems that have no name or that are inaccessible.
-		if(it.first.empty() || other.Name().empty() || other.Inaccessible())
+		if(it.first.empty() || other.trueName.empty() || other.Inaccessible())
```

Comment:
```suggestion
		if(it.first.empty() || other.TrueName().empty() || other.Inaccessible())
```
It was using the getter before.

Reply:
~~which doesn't mean it needs to use it~~

Reply:
It probably gets inlined anyway, so let's keep using the accessor for now.